### ISSUE TITLE
[🐛] No permission popup shown in Android

### ISSUE THREAD

**Issue by zhouhao27** [react-native-image-picker/react-native-image-picker#2199]
### Description

When I call `launchImageLibrary` or `launchCamera`, there is no permission popup.

### How to repeat issue and example

- step to reproduce the issue
- ...

### Solution

What needs to be done to address this issue?

### Additional Information

- Image Picker version: 4.0.6
- React Native version: 0.65.1
- Platform: Android
- Development Operating System: MacOS
- Dev tools: Android Studio Giraffe | 2022.3.1 Patch 1

**Comment by garawalid** [comment]
You need to request the permission yourself.
```javascript
PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.READ_MEDIA_IMAGES).then((permissionStatus) => {
        if (permissionStatus === PermissionsAndroid.RESULTS.GRANTED) {
            console.log("User granted permissions request ");
        } else {
            console.log("User denied permissions request");
        }
```
And you need to add the following permission to **AndroidManifest.xml**
```xml
<uses-permission android:name="android.permission.READ_MEDIA_IMAGES"/>
```

**Comment by Yandamuri** [comment]
Try version `4.9.0` as there some changes pertaining to `Android 13`

Check this -- https://github.com/react-native-image-picker/react-native-image-picker/pull/1955

**Comment by Yandamuri** [comment]
@garawalid `READ_MEDIA_IMAGES` is introduced in `Android 13`. Does it mean no permission is required below `Android 13` ?

**Comment by garawalid** [comment]
> @garawalid `READ_MEDIA_IMAGES` is introduced in `Android 13`. Does it mean no permission is required below `Android 13` ?

Exactly, you don't need to request this permission for  Android 12 and below.

**Comment by Yandamuri** [comment]
@garawalid I am using `react-native-image-picker: 3.1.4`. I am able to pic images from gallery in `Android 13` without using `READ_MEDIA_IMAGES` permission. Any specific version of  `react-native-image-picker` handles `Android 13` changes?
